src/schema/v2/input_fields/partner_category_type.ts

```diff
@@ -1,8 +1,8 @@
 import { GraphQLEnumType } from "graphql"
 
-const CategoryType = {
+const PartnerCategoryTypeEnum = {
```

Comment:
Should enumerated types now always be appended with `...Enum`?

Reply:
It was either `PartnerCategoryTypeType` or `PartnerCtegoryTypeEnum`. 😄. We actually _do_ have a precedent of appending Enum to the end of an enum type. It doesn't actually effect the name that's in the schema though.

Reply:
Probably a good idea yeah, so this needs to be added to the design doc and have a lint/peril rule.